structuredheadings/plugin.js

```diff
@@ -47,19 +107,34 @@
          "matchheading," +
          "increaseheadinglevel," +
          "decreaseheadinglevel," +
-         "restartNumbering",
+         "restartNumbering," +
+         "selectStyle",
+    hidpi: true,
     init: function (editor) {
       editor.addContentsCss(this.path + "styles/numbering.css");
-      editor.addCommand("autoNumberHeading",
-        CKEDITOR.plugins.structuredheadings.commands.autoNumberHeading);
-      editor.addCommand("matchHeading",
-        CKEDITOR.plugins.structuredheadings.commands.matchHeading);
-      editor.addCommand("increaseHeadingLevel",
-        CKEDITOR.plugins.structuredheadings.commands.increaseHeadingLevel);
-      editor.addCommand("decreaseHeadingLevel",
-        CKEDITOR.plugins.structuredheadings.commands.decreaseHeadingLevel);
-      editor.addCommand("restartNumbering",
-        CKEDITOR.plugins.structuredheadings.commands.restartNumbering);
+
+      //Wrap addCommands in an iffy to get around lint max-statements in init
+      (function () {
+        editor.addCommand("autoNumberHeading",
+          CKEDITOR.plugins.structuredheadings.commands.autoNumberHeading);
+        editor.addCommand("matchHeading",
+          CKEDITOR.plugins.structuredheadings.commands.matchHeading);
+        editor.addCommand("increaseHeadingLevel",
+          CKEDITOR.plugins.structuredheadings.commands.increaseHeadingLevel);
+        editor.addCommand("decreaseHeadingLevel",
+          CKEDITOR.plugins.structuredheadings.commands.decreaseHeadingLevel);
+        editor.addCommand("restartNumbering",
+          CKEDITOR.plugins.structuredheadings.commands.restartNumbering);
+        editor.addCommand("setCurrentStyle",
+                CKEDITOR.plugins.structuredheadings.commands.setCurrentStyle);
+        editor.addCommand("reapplyStyle",
+          CKEDITOR.plugins.structuredheadings.commands.reapplyStyle);
+      })();
```

Comment:
Nicer stacktrace + organization mainly.  This would be a bit silly given other test coverage but you could also test the command registration by itself that way.